docs/users_guide/admin_index.md

```diff
@@ -514,56 +514,109 @@ An admin user can view their profile as if they were an ordinary user on [this d
+    **Email description:** Emails instructor to inform them they have not been assigned to a workshop    
+    **Sent to:**  Instructor  
+    **Date sent:** One hour after conditions are met     
+    **Conditions:** Instructor declined in recruitment process  
+
+??? abstract "Instructor Signs Up for Workshop"
+    [View in AMY](https://amy.carpentries.org/emails/template/43754daa-c312-41db-a0f6-00ac4f88bafd/)  
+    **Email description:** Emails instructor to confirm they have expressed interest in a workshop   
+    **Sent to:**  Instructor  
+    **Date sent:** One hour after conditions are met     
+    **Conditions:** Instructor signs up in recruitment process  
+
+??? abstract "New Self-Organised Workshop"
+    [View in AMY](https://amy.carpentries.org/emails/template/f967122e-55a8-483e-a571-c3dff43dd33e/)  
+    **Email description:** Emails Self-Organised workshop host to confirm their submission has been accepted  
+    **Sent to:**  Workshop Host  
+    **Date sent:** One hour after conditions are met     
+    **Conditions:** Self organized submission form accepted
+
+??? abstract "Post Workshop 7 days"
+    [View in AMY](https://amy.carpentries.org/emails/template/4809378c-c1aa-4118-a3a7-cbc9586c686b/)  
+    **Email description:** Follow up one week after Centrally-Organised workshop is complete  
+    **Sent to:** Hosts and Instructors     
+    **Date sent:** Later of seven days after workshop end or seven days from now      
+    **Conditions:** Centrally-Organised; At least one host; At least one Instructor;  Tagged SWC/DC/LC; 
+
+??? abstract "Recruit helpers"
+    [View in AMY](https://amy.carpentries.org/emails/template/d1b20e74-a245-4c7d-9182-663f3acd8a48/)
+    **Email description:** Reminder to host to recruit helpers
+    **Sent to:** Hosts and Instructors     
+    **Date sent:** 21 days before event  
+    **Conditions:** Centrally-Organised; At least one host; At least 1 Instructor;  Tagged SWC/DC/LC; Not stalled/cancelled; At least one host; At least one Instructor; No helpers; Start date at least 14 days in future
+
+
+
+#### Active emails (Instructor Training)
+
+??? abstract "Instructor Training (One month away)"
+    [View in AMY](https://amy.carpentries.org/emails/template/bafe53c4-e486-4a6e-8b5d-894790946913/)  
+    **Email description:** Send Trainers information about upcoming Instructor Training  
+    **Sent to:** Trainers  
+    **Date sent:** One month before event start  
+    **Conditions:** Tag TTT; At least 2 Trainers; Start date in future  
+
+#### Active emails (Memberships)
+
+??? abstract "New / renewing membership starting (member onboarding)"
+    [View in AMY](https://amy.carpentries.org/emails/template/f1311aaf-c546-429b-930c-c36a5a8009fb/)  
+    **Email description:**  Onboarding message for new/renewing memberships 
+    **Sent to:**   Programmatic and billing contacts
+    **Date sent:** One month before membership start date or immediately (whichever is first)   
```

Comment:
Currently coded as: One month before membership start or immediately - whichever is **later**.